### PR TITLE
Initial jest tests for event create input change

### PR DESCRIPTION
**Pull request by ekong2**

### CODE REVIEW

File: client/test/__tests__/EventCreateValues.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+var React = require('react/addons');
+var TestUtils = React.addons.TestUtils;
+
+var EventCreate = require('../../public/js/components/EventCreate.js');
+
+describe('Successfully update values on Form', function() {
+  it('Should update value', function() {
+    var React = require('react/addons');
+    var EventCreate = require(componentPath + 'EventCreate.js');
+    var TestUtils = React.addons.TestUtils;
```
Comment (kevhuang) [inline client/test/__tests__/EventCreateValues.js:8]: Are these duplications needed? They were previously defined in lines 4-7.

**Comment by kevhuang** [comment]
:+1: Maybe get rid of the duplications I mentioned, and then close this pull request and submit a new one to see if Travis will build successfully

File: client/test/__tests__/EventCreateValues.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+var componentPath = '../../public/js/components/'
```
Comment (kevhuang) [inline client/test/__tests__/EventCreateValues.js]: missing semi-colon at the end

**Comment by dmsakamoto** [comment]
:+1:

**Comment by stevenshuhyow** [comment]
:+1:

**Pull request merged**